todoman/model.py

```diff
@@ -306,17 +306,14 @@ def from_file(cls, path, id=None):
         with open(path, 'rb') as f:
             cal = f.read()
             cal = icalendar.Calendar.from_ical(cal)
-            try:
-                component = cal.walk('VTODO')[0]
+            for component in cal.walk('VTODO'):
```

Comment:
I'd rather `raise` in this method, a bit like  #180.

Also, I'd rather not support more than one-per-file by default, since this is vdir-incompliant (and vdirsyncer-incompatible).